libraries/cyclestreets-core/src/main/java/net/cyclestreets/api/client/RetrofitApiClient.java

```diff
@@ -146,13 +148,22 @@ public String getJourneyJson(final String plan,
                                final String leaving,
                                final String arriving,
                                final int speed) throws IOException {
-    Response<String> response = v1Api.getJourneyJson(plan, itineraryPoints, leaving, arriving, speed).execute();
+    Response<String> response = v1Api.getJourneyJson(plan, itineraryPoints, leaving, arriving, speed, REPORT_ERRORS).execute();
```

Comment:
> the waypoints that are generated for a circular route being displayed on the map

Just to say that this isn't actually necessary for a circular leisure route. It's arguably perhaps an error that the API actually transmits waypoints other than start/finish waypoint.

Reply:
But they are needed if a replan becomes necessary in LiveRide, if I remember correctly.

Reply:
First 2 screenshots show LiveRide for a circular route.  3rd screenshot shows replan after going off route:
<img width="737" alt="LiveRide replan" src="https://user-images.githubusercontent.com/59799824/112892620-732fa900-90d1-11eb-9690-14bbcefca385.PNG">

Reply:
Yes, that's true, so yes, I suppose we do need to transmit them.

However, the point remains that they don't need to be shown in the general route result, as they're really 'virtual' waypoints, since the user never set them.

Reply:
OK, fine by me to leave out the waypoints then 👍 

btw I've just had a play with the circular route UI on my phone since the upgrade went out after I merged this, looking good.  Great contribution @HilaryN, thank you again.  I promise to respond promptly once you've had the chance to get the translations prepared!